OutOfSchool/OutOfSchool.WebApi/Common/Utilities/PictureManager.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Drawing;
+using System.Drawing.Drawing2D;
+using System.Drawing.Imaging;
+using System.IO;
+using System.Linq;
+
+namespace OutOfSchool.WebApi.Common.Utilities
+{
+    public class PictureManager
+    {
+        private const long Quality = 75;
```

Comment:
maybe extract to options/appsettings too? and it's better to have `90`% quality. As for HD jpeg with 75% we'll be saving only around 50 KB space, but with substantial loss of quality compared to 90%.